app/src/main/java/com/github/se/wanderpals/model/viewmodel/SuggestionsViewModel.kt

```diff
@@ -106,4 +108,24 @@ open class SuggestionsViewModel(
       }
     }
   }
+
+  open fun addComment(tripId: String, suggestion: Suggestion, comment: Comment) {
+    val updatedSuggestion =
+        suggestion.copy(
+            comments =
+                suggestion.comments +
+                    Comment(
+                        commentId = UUID.randomUUID().toString(),
+                        userId = currentLoggedInUId,
+                        userName = comment.userName,
+                        text = comment.text,
+                        createdAt = comment.createdAt))
+    viewModelScope.launch {
+      val wasUpdateSuccessful =

```

Comment:
Creating a variable isn't really needed here as I understand it, but it makes sense since toggleLikeSuggestion also uses it